R/callOpenTiles.R

```diff
@@ -375,9 +375,16 @@ setMethod(
   # of interest
   if (all(cellPopulations == "ALL")) {
     cellPopulations <- colnames(allCellCounts)
-  } else {
+  } else if(any(rownames(allCellCounts) %in% cellPopulations)){
     allCellCounts <- allCellCounts[rownames(allCellCounts) %in% cellPopulations, , drop = FALSE]
     allFragmentCounts <- allFragmentCounts[rownames(allFragmentCounts) %in% cellPopulations, , drop = FALSE]
+  } else{
+  
+    stop(
+      stringr::str_interp("Some or all of the cell populations provided were not found in the metadata column ${cellPopLabel}. 
+        These cell populations include ${cellPopulations}.")
+    )
```

Comment:
This kind of check should happen earlier, like in to the top of `.callOpenTiles_default`.

Reply:
It should: check that all of `cellPopulations` exists in `unique(cellColData[[cellPopLabel]])`

Reply:
It does occur in .callOpenTiles_default, but .callOpenTiles_default isn't run when we use an ArchR project.

Reply:
Hence the need to check within .callOpenTiles_ArchR